utils/vlog/vlog_to_model.py

```diff
@@ -151,7 +153,7 @@ def is_registered_path(tmod, pin, pout):
             continue
         deps_files.add(im.group(1))
 
-if True:
+if len(deps_files) > 0:
```

Comment:
this condition is unnecessary. If `len(deps_files)` == 0 the bellow for would not execute.

Reply:
@kgugala As you are currently not emitting the correct output in `pb_type` vpr complains about the model file. Lets only emit the leaf nodes for now.

Reply:
@mithro the `pb_type` seems to be correct. It's the model which is not, but the above change is actually fixing it.

In the `multiple_instance` test the `multiple_instance` pb_type is not a primitive (it includes counter primitive), so it should not have a model. The only fix necessary here is to update `golden.model.xml` for the `multiple_instance` test. This can be done by calling 
`make multiple_instance_update_golden_model` in the `build/utils/vlog/tests/multiple_instance` directory

Reply:
Now I understand your confusion, I was sure I had already pushed a fix for this... Done now.